net/p0f/Portfile

```diff
@@ -17,13 +17,12 @@ long_description    p0f is a versatile passive OS fingerprinting and \
                     information gathering on servers, firewalls, IDSes, and \
                     honeypots, for pen-testing, or just for the fun of it.
 
-homepage            http://lcamtuf.coredump.cx/p0f3/
-master_sites        ${homepage}releases/ \
-                    ${homepage}releases/old/${major}.x/
+homepage            http://lcamtuf.coredump.cx/p0f3
```

Comment:
Why removing the trailing `/`?

```
$ curl -I 'http://lcamtuf.coredump.cx/p0f3'
HTTP/1.1 301 Moved Permanently
Date: Wed, 08 Nov 2017 07:52:30 GMT
Server: Apache
Location: http://lcamtuf.coredump.cx/p0f3/
Content-Type: text/html; charset=iso-8859-1
```

Reply:
updated. Leaving the double // in the master_sites since its easier to read